pkg/apis/mattermost/v1alpha1/clusterinstallation_types.go

```diff
@@ -180,6 +180,10 @@ type Database struct {
 	BackupRemoteDeletePolicy string `json:"backupRemoteDeletePolicy,omitempty"`
 	// Defines the secret to be used for uploading/restoring backup.
 	// +optional
+	BackupSecretName string `json:"backupSecretName,omitempty"`
+	// Optionally enter the name of already existing Secret
```

Comment:
We may want to be more specific here on the description as we already have a few secrets that can be defined in the spec. Maybe something like:
```suggestion
	// Optionally enter the name of an already existing Secret for use by the MySQL operator
```